### PR TITLE
Add Annotation Version fo Contact Entity to README.md

### PR DESCRIPTION
**Pull request by rkueny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,33 @@ Acme\ContactBundle\Entity\Contact:
...
+use Mremi\ContactBundle\Entity\Contact as BaseContact;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="contact")
+ */
+class Contact extends BaseContact
+{
+    /**
+     * @var integer
```
Comment (mremi) [inline README.md]: Should be `int`, can you also fix it above please? Please squash your commits (or amend the existing one) to have only one. Thx

Comment (rkueny) [inline README.md]: I can use `int` if you want but in your file you use `@var integer` and Symfony use `@var integer` too

Comment (mremi) [inline README.md]: Which file? You can see the naming conventions of Symfony here http://symfony.com/doc/current/contributing/code/standards.html#naming-conventions

Comment (rkueny) [inline README.md]: https://github.com/mremi/ContactBundle/edit/master/README.md#l117

I'll update

Comment (mremi) [inline README.md]: Yep this is what I mentioned in my first comment, thank you!

**Pull request closed without merging**